accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -134,10 +445,72 @@ Windows-specific:
     - `set_PriorityLevel(ThreadPriorityLevel)`
     - `set_ProcessorAffinity(IntPtr)`
 
+### System.Drawing
+
+* **FontConverter**
+    - `.ctor()`
+    - `CanConvertFrom(ITypeDescriptorContext, Type)`
+    - `CanConvertTo(ITypeDescriptorContext, Type)`
+    - `ConvertFrom(ITypeDescriptorContext, CultureInfo, Object)`
+    - `ConvertTo(ITypeDescriptorContext, CultureInfo, Object, Type)`
+    - `CreateInstance(ITypeDescriptorContext, IDictionary)`
+    - `GetCreateInstanceSupported(ITypeDescriptorContext)`
+    - `GetProperties(ITypeDescriptorContext, Object, Attribute[])`
+    - `GetPropertiesSupported(ITypeDescriptorContext)`
+* **FontConverter+FontNameConverter**
+    - `.ctor()`
+    - `CanConvertFrom(ITypeDescriptorContext, Type)`
+    - `ConvertFrom(ITypeDescriptorContext, CultureInfo, Object)`
+    - `GetStandardValues(ITypeDescriptorContext)`
+    - `GetStandardValuesExclusive(ITypeDescriptorContext)`
+    - `GetStandardValuesSupported(ITypeDescriptorContext)`
+* **FontConverter+FontUnitConverter**
+    - `.ctor()`
+    - `GetStandardValues(ITypeDescriptorContext)`
+* **IconConverter**
+    - `.ctor()`
+    - `CanConvertFrom(ITypeDescriptorContext, Type)`
+    - `CanConvertTo(ITypeDescriptorContext, Type)`
+    - `ConvertFrom(ITypeDescriptorContext, CultureInfo, Object)`
+    - `ConvertTo(ITypeDescriptorContext, CultureInfo, Object, Type)`
+* **ImageConverter**
```

Comment:
Shouldn't this be supported?

Reply:
@safern @tannergooding

Reply:
@safern @tannergooding

Reply:
I believe most of System.Drawing should be supported; but it depends on what `libgdiplus` currently implements as I believe it has a few differences

Reply:
ImageConverter currently lives in `System.Windows.Extensions`, mainly to avoid bringing TypeConverter into common drawing scenarios, but we have an issue to fix that: https://github.com/dotnet/runtime/issues/1011

So we should probably remove the drawing converters from this list.  cc @safern

Reply:
I agree. It is in our 5.0.0 priorities to move these converters down to System.Drawing.Common as they're definitely supported on non-windows.

Reply:
Done